src/wallet/rpcwallet.cpp

```diff
@@ -2779,11 +2779,14 @@ static RPCHelpMan upgradetohd()
 {
     return RPCHelpMan{"upgradetohd",
         "\nUpgrades non-HD wallets to HD.\n"
+        "\nThe wallet passphrase must be supplied as it is used to encrypt the HD chain. Unlocking the wallet beforehand"
```

Comment:
is it breaking changes? So, we can't more create HD wallet from plank wallet without passphrase?

Reply:
Nope, you can still create HD wallets from blank wallets and can still encrypt them by supplying a passphrase but I can see how the phrasing would imply otherwise. Will update after CI results.